src/components/util/RivetIcons.jsx

```diff
@@ -38,33 +24,6 @@ const IconCore = ({ children, ...attrs }) => {
   );
 };
 
-const icoBell = (attrs) => (
-  <IconCore data-testid={TestUtils.RivetIcons.testId} {...attrs}>
-    <path
-      fill="currentColor"
-      d="M14.57,12.06,13,9.7V6A5,5,0,0,0,3,6V9.7L1.43,12.06a1.25,1.25,0,0,0,1,1.94H6a2,2,0,0,0,4,0h3.53a1.25,1.25,0,0,0,1-1.94ZM8,12H3.87L5,10.3V6a3,3,0,0,1,6,0v4.3L12.13,12Z"
-    />
-  </IconCore>
-);
-
-const icoCaretDown = (attrs) => (
-  <IconCore data-testid={TestUtils.RivetIcons.testId} {...attrs}>
-    <path
-      fill="currentColor"
-      d="M8,12.46a2,2,0,0,1-1.52-.7L1.24,5.65a1,1,0,1,1,1.52-1.3L8,10.46l5.24-6.11a1,1,0,0,1,1.52,1.3L9.52,11.76A2,2,0,0,1,8,12.46Z"
-    />
-  </IconCore>
-);
-
-const icoFile = (attrs) => (
-  <IconCore {...attrs}>
-    <path
-      fill="currentColor"
-      d="M10.41,1H3.5A1.3,1.3,0,0,0,2.2,2.3V13.7A1.3,1.3,0,0,0,3.5,15h9a1.3,1.3,0,0,0,1.3-1.3V4.39ZM11.8,5.21V6H9.25V3h.34ZM4.2,13V3h3V6.75A1.25,1.25,0,0,0,8.5,8h3.3v5Z"
-    />
-  </IconCore>
-);
-
 const icoLogo = (attrs) => (
```

Comment:
I'm unsure if this would work well in this case, but you could try using Rivet Icon's [`registerIcon()` utility](https://github.com/indiana-university/rivet-icons?tab=readme-ov-file#registericon-function) to add custom icons. That would allow you to do something like `<rvt-icon name="trident-header" />`.